app/src/components/players/PlayerOverviewGroups.tsx

```diff
@@ -21,7 +21,7 @@ export async function PlayerOverviewMemberships(props: PlayerOverviewMemberships
 
   return (
     <div>
-      <Label className="text-xs leading-4 text-gray-200">Group affilitations</Label>
+      <Label className="text-xs leading-4 text-gray-200">Group affiliations</Label>
```

Comment:
I can't believe I haven't noticed this typo in over 1.5 years that has been up 🤦  thanks for this